### ISSUE TITLE
No error when non-TypedDict base precedes TypedDict

### ISSUE THREAD

**Issue by DanielRosenwasser** [microsoft/pyright#5937]
```py
from typing import TypedDict

class Person:
    name: str
    age: int

class PersonDict1(TypedDict, Person):
    #                        ~~~~~~
    # error: All base classes for TypedDict classes must also be TypedDict classes
    pass

class PersonDict2(Person, TypedDict, object):
    #                                ~~~~~~
    # error: All base classes for TypedDict classes must also be TypedDict classes
    pass

class PersonDict3(Person, TypedDict):
    # no error!
    pass

```

I would expect an error on all of these examples, but the last one is missing an error.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.327, which I just published. It will also be included in an upcoming release of pylance.
